pkg/cmd/pulumi/package_gen_sdk.go

```diff
@@ -82,25 +80,85 @@ func newGenSdkCommand() *cobra.Command {
 }
 
 func genSDK(language, out string, pkg *schema.Package, overlays string) error {
-	var f func(string, *schema.Package, map[string][]byte) (map[string][]byte, error)
+	cwd, err := os.Getwd()
+	if err != nil {
+		return fmt.Errorf("get current working directory: %w", err)
+	}
+
+	var f func(string, *schema.Package, map[string][]byte) error
```

Comment:
Can you provide some colour into the purpose for this variable? The name isn't particularly descriptive, so a comment would help with (my) comprehension.

Reply:
Renamed